hooks/useWallet.ts

```diff
@@ -0,0 +1,68 @@
+import { JsonRpcSigner, Web3Provider } from '@ethersproject/providers'
+import { useWeb3React } from '@web3-react/core'
+import { Web3ReactManagerFunctions } from '@web3-react/core/dist/types'
+import { useCallback, useEffect, useState } from 'react'
+import { injectedConnector } from '../lib/connector'
+
+export default function useWallet(): {
```

Comment:
should have the `key?: string` as required by `useWeb3React` to support multiple instances of web3react